### PR TITLE
jpegoptim: added max size param example

### PR DESCRIPTION
**Pull request by nutellinoit**
----

Hi again, I find it useful to have an example of recursive optimization of `jpegoptim`
I use it only this way , also with --size param

`find {{.}} -name "{{*.jpg}}" -exec jpegoptim --strip-all --all-progressive --size={{250k}}  {} \;`

### CODE REVIEW

File: pages/common/jpegoptim.md [hunk 1]
```diff
@@ -13,3 +13,11 @@
 - Force the output images to be progressive:
 
 `jpegoptim --all-progressive {{image1.jpeg}} {{image2.jpeg}} {{imageN.jpeg}}`
+
+- Force the output images to be fixed max size:
+
+`jpegoptim --size={{250k}} {{image1.jpeg}} {{image2.jpeg}} {{imageN.jpeg}}`
+
+- Recursive folder optimization:
```
Comment (agnivade) [inline pages/common/jpegoptim.md]: Sorry, but we do not usually mention examples like these. It is out of scope for tldr. This falls into the category of cheat sheets.

Rarely we might consider one or two exceptions, but I think this is fairly straightforward to construct, given that `tldr find` already shows the -exec usage.

Comment (sbrl) [reply pages/common/jpegoptim.md]: I'm inclined to agree here.

If you're curious for an alternative way of doing that that maxes out all CPU cores though, I've got this one-liner stashed away:

```
find . -regextype egrep -iregex '^.*\.jpe?g$' -print0 | xargs -0 -n1 -P4 jpegoptim --preserve --all-progressive
```

Comment (nutellinoit) [reply pages/common/jpegoptim.md]: @sbrl thank you! 

i changed the page only with --size line

File: pages/common/jpegoptim.md [hunk 2]
```diff
@@ -13,3 +13,7 @@
 - Force the output images to be progressive:
 
 `jpegoptim --all-progressive {{image1.jpeg}} {{image2.jpeg}} {{imageN.jpeg}}`
+
+- Force the output images to be fixed max size:
```
Comment (agnivade) [inline pages/common/jpegoptim.md]: I think "fixed upto a max size" is grammatically correct.

Comment (sbrl) [reply pages/common/jpegoptim.md]: Perhaps `to have a fixed maximum size` would read better?

**Pull request merged**